docs/modules/rediscache.md

```diff
@@ -28,6 +28,13 @@ Provisions a new Redis cache.
 | `enableNonSslPort ` | `string`            | Specifies whether the non-SSL Redis server port (6379) is enabled. Valid values: (`enabled`, `disabled`) | N                                                            | If not provided, `disabled` is used. That is, you can't use non-SSL Redis server port by default. |
 | `tags`              | `map[string]string` | Tags to be applied to new resources, specified as key/value pairs. | N                                                            | Tags (even if none are specified) are automatically supplemented with `heritage: open-service-broker-azure`. |
 
+For `premium` plan, following provisioning parameter is available:
+| Parameter Name                | Type     | Description                                                  | Required                                             | Default Value                                                |
```

Comment:
IIRC, a blank line is required above the table.

Reply:
Thanks, I'll also fix other PRs.